packages/app/src/cli/services/dev/extension/server/utilities.ts

```diff
@@ -52,6 +52,15 @@ export function getExtensionPointRedirectUrl(
       rawUrl.searchParams.append('url', getExtensionUrl(extension, options))
       rawUrl.searchParams.append('target', requestedTarget)
       break
+    case 'customer-account':
+      const [storeName, ...storeDomainParts] = options.storeFqdn.split('.')
```

Comment:
Can we clean this up to share code with the legacy path on line 22?

Reply:
Thanks. Since CLA failed in this pr, I will close this pr and use https://github.com/Shopify/cli/pull/2728 intead.